requirements.txt

```diff
@@ -1,3 +1,4 @@
+backports.shutil_get_terminal_size==1.0.0
```

Comment:
I think you'll need to add this to `setup.py` too so that it's included when people do a `pip install`